docs/content/en/docs/cluster-operator/vms-deploy.md

```diff
@@ -2,14 +2,22 @@
 title: "Deploy Cluster on VMs"
 linkTitle: "Deploy Cluster on VMs"
 description: >
-    The easiest way to deploy cluster on VMs with Kurator.
+  The easiest way to deploy cluster on VMs with Kurator.
 ---
 
 You can easily manage the VMs cluster with Kurator, including the installation, deletion, upgrade and scale of the VMs cluster.
 
 These properties are built on [Cluster API](https://cluster-api.sigs.k8s.io) and [KubeSpray](https://kubespray.io/).
 
-This guide will describe how to create and delete the k8s cluster on VMs with Kurator.
+This guide will describe how to manage the k8s cluster on VMs with Kurator.
+
+This guide is mainly composed of the following parts：
+
+- [Prerequisites](#prerequisites)
+- [Deploy a k8s cluster on VMs](#deploy-a-k8s-cluster-on-vms)
+- [Scale nodes on VMs](#scale-nodes-on-vms)
```

Comment:
do we really need this part?
`docsy` will auto generated sidebar.

Reply:
This is a jump within a single page, which is different from docsy sidebar. 
The effect of this is similar to: https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/tools/install-kubectl-linux.md#install-kubectl-binary-with-curl-on-linux.
Because there is too much content on a single page, adding this can facilitate jumping to what user need.

Reply:
![image](https://user-images.githubusercontent.com/4354057/224266588-87e97a4d-fcb4-4243-ab74-324e1b1e7052.png)

Reply:
you are right，I didn't notice this feature